app/templates/play/ladder_home.jade

```diff
@@ -23,5 +23,11 @@ block content
               if playCount
                 span.spl.spr - #{playCount.sessions}
                 span(data-i18n="play.players") players
+                if (view.levelStatusMap[level.id]=='complete')
+                  span.spl.spr -
+                  span(class="level-status-#{view.levelStatusMap[level.id]}", data-i18n="clans.complete_2") Complete
+                else if (view.levelStatusMap[level.id]=='started')
+                  span.spl.spr -
+                  span(class="level-status-#{view.levelStatusMap[level.id]}", data-i18n="clans.started_2") Started
```

Comment:
One last thing! Remove the `Complete` and `Started` strings from the ends of these lines, since they're redundant with the translation and we can visually check that the translations are hooked up correctly (if they aren't, nothing shows up).